src/Runner.Listener/Runner.cs

```diff
@@ -252,7 +254,7 @@ public async Task<int> ExecuteCommand(CommandSettings command)
                     }
                 }
 
-                RunnerSettings settings = configManager.LoadSettings();
+                _runnerSettings = configManager.LoadSettings();
```

Comment:
make this a global variable, so we can reuse it later (next PR) without reloading it from disk.

Reply:
nit: _instance_ not _global_

Reply:
Many methods throughout this class receive `RunnerSettings` as a parameter. Will those all switch to use the instance variable instead?

Reply:
nvm, i am going to switch back, the instance var might cause more changes...